Script/AtomicEditor/editor/Editor.ts

```diff
@@ -138,6 +138,41 @@ class Editor extends Atomic.ScriptObject {
         return true;
     }
 
+    /**
+     * Return a preference value or the provided default from the user settings file
+     * @param  {string} extensionName name of the extension the preference lives under
+     * @param  {string} preferenceName name of the preference to retrieve
+     * @param  {number | boolean | string} defaultValue value to return if pref doesn't exist
+     * @return {number|boolean|string}
+     */
```

Comment:
I actually use https://atom.io/packages/docblockr .  It stubs out the doc comments when you type ```/**``` and you just have to fill in the details.  Very handy.